src/main/java/com/gluonhq/substrate/util/ios/Deploy.java

```diff
@@ -83,7 +84,7 @@ private void checkPrerequisites() throws IOException, InterruptedException {
         }
 
         // Check for libimobiledevice installed
-        List<String> libiPath = checkDependencyPaths("libimobiledevice");
+        List<String> libiPath = checkDependencyPaths(LIBIMOBILEDEVICE);
         if (libiPath == null || libiPath.isEmpty() || !Files.exists(Path.of(libiPath.get(0)))) {
             Logger.logSevere("Error finding libimobiledevice.dylib");
```

Comment:
Perhaps we should no longer mention .dylib here, but just inform the user that the library could not be found?

Reply:
In fact, this check is redundant, `checkDependencyPaths` does it too, and will throw already a RTE. I've done a small refactoring.